From 18713d582b94c7f132f306c01c43fd22a12a1fc5 Mon Sep 17 00:00:00 2001
From: Radek Polak <psonek2@seznam.cz>
Date: Fri, 2 Oct 2009 20:54:29 +0200
Subject: [PATCH 2/7] GTA01/GTA02: disable android drivers in default configs

Android drivers are not needed for standard linux systems. We can save
memory by disabling them.

Android low memory killer (CONFIG_ANDROID_LOW_MEMORY_KILLER) also
prevents using swap by killing applications before swapping can start
on non android systems.

Reported-by: Jim Morris <morris@wolfman.com>
Signed-off-by: Radek Polak <psonek2@seznam.cz>
Signed-off-by: Nelson Castillo <arhuaco@freaks-unidos.net>
---
 arch/arm/configs/gta01_moredrivers_defconfig |   21 ++++++++++-----------
 arch/arm/configs/gta02_moredrivers_defconfig |   19 ++++++++-----------
 arch/arm/configs/gta02_packaging_defconfig   |   19 ++++++++-----------
 3 files changed, 26 insertions(+), 33 deletions(-)

diff --git a/arch/arm/configs/gta01_moredrivers_defconfig b/arch/arm/configs/gta01_moredrivers_defconfig
index 411acab..68b95e7 100644
--- a/arch/arm/configs/gta01_moredrivers_defconfig
+++ b/arch/arm/configs/gta01_moredrivers_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.29-rc3
-# Tue Feb 24 02:13:21 2009
+# Fri Oct  2 16:49:49 2009
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -203,6 +203,7 @@ CONFIG_S3C_GPIO_SPACE=0
 CONFIG_S3C_GPIO_TRACK=y
 CONFIG_S3C_DMA=y
 CONFIG_S3C_PWM=y
+CONFIG_S3C_DEV_USB_HOST=y
 
 #
 # S3C2400 Machines
@@ -1159,6 +1160,7 @@ CONFIG_WATCHDOG=y
 #
 # CONFIG_SOFT_WATCHDOG is not set
 CONFIG_S3C2410_WATCHDOG=m
+CONFIG_PCF50606_WATCHDOG=y
 
 #
 # USB-based Watchdog Cards
@@ -1686,17 +1688,14 @@ CONFIG_STAGING=y
 #
 # Android
 #
-CONFIG_ANDROID=y
-CONFIG_ANDROID_BINDER_IPC=y
-CONFIG_ANDROID_LOGGER=y
-CONFIG_ANDROID_RAM_CONSOLE=y
-CONFIG_ANDROID_RAM_CONSOLE_ENABLE_VERBOSE=y
-# CONFIG_ANDROID_RAM_CONSOLE_ERROR_CORRECTION is not set
-# CONFIG_ANDROID_RAM_CONSOLE_EARLY_INIT is not set
-CONFIG_ANDROID_TIMED_GPIO=y
-CONFIG_ANDROID_LOW_MEMORY_KILLER=y
+# CONFIG_ANDROID is not set
+# CONFIG_ANDROID_BINDER_IPC is not set
+# CONFIG_ANDROID_LOGGER is not set
+# CONFIG_ANDROID_RAM_CONSOLE is not set
+# CONFIG_ANDROID_TIMED_GPIO is not set
+# CONFIG_ANDROID_LOW_MEMORY_KILLER is not set
 # CONFIG_ANDROID_WAKELOCK is not set
-CONFIG_ANDROID_PARANOID_NETWORK=y
+# CONFIG_ANDROID_PARANOID_NETWORK is not set
 
 #
 # File systems
diff --git a/arch/arm/configs/gta02_moredrivers_defconfig b/arch/arm/configs/gta02_moredrivers_defconfig
index 13b4121..e4fe9a9 100644
--- a/arch/arm/configs/gta02_moredrivers_defconfig
+++ b/arch/arm/configs/gta02_moredrivers_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.29-rc3
-# Sun Apr 19 23:22:38 2009
+# Fri Oct  2 20:52:45 2009
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -1782,17 +1782,14 @@ CONFIG_STAGING=y
 #
 # Android
 #
-CONFIG_ANDROID=y
-CONFIG_ANDROID_BINDER_IPC=y
-CONFIG_ANDROID_LOGGER=y
-CONFIG_ANDROID_RAM_CONSOLE=y
-CONFIG_ANDROID_RAM_CONSOLE_ENABLE_VERBOSE=y
-# CONFIG_ANDROID_RAM_CONSOLE_ERROR_CORRECTION is not set
-# CONFIG_ANDROID_RAM_CONSOLE_EARLY_INIT is not set
-CONFIG_ANDROID_TIMED_GPIO=y
-CONFIG_ANDROID_LOW_MEMORY_KILLER=y
+# CONFIG_ANDROID is not set
+# CONFIG_ANDROID_BINDER_IPC is not set
+# CONFIG_ANDROID_LOGGER is not set
+# CONFIG_ANDROID_RAM_CONSOLE is not set
+# CONFIG_ANDROID_TIMED_GPIO is not set
+# CONFIG_ANDROID_LOW_MEMORY_KILLER is not set
 # CONFIG_ANDROID_WAKELOCK is not set
-CONFIG_ANDROID_PARANOID_NETWORK=y
+# CONFIG_ANDROID_PARANOID_NETWORK is not set
 
 #
 # File systems
diff --git a/arch/arm/configs/gta02_packaging_defconfig b/arch/arm/configs/gta02_packaging_defconfig
index 03c72d5..e2dbbef 100644
--- a/arch/arm/configs/gta02_packaging_defconfig
+++ b/arch/arm/configs/gta02_packaging_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.29-rc3
-# Sun Apr 19 22:57:25 2009
+# Fri Oct  2 20:51:57 2009
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -1785,17 +1785,14 @@ CONFIG_STAGING=y
 #
 # Android
 #
-CONFIG_ANDROID=y
-CONFIG_ANDROID_BINDER_IPC=y
-CONFIG_ANDROID_LOGGER=y
-CONFIG_ANDROID_RAM_CONSOLE=y
-CONFIG_ANDROID_RAM_CONSOLE_ENABLE_VERBOSE=y
-# CONFIG_ANDROID_RAM_CONSOLE_ERROR_CORRECTION is not set
-# CONFIG_ANDROID_RAM_CONSOLE_EARLY_INIT is not set
-CONFIG_ANDROID_TIMED_GPIO=y
-CONFIG_ANDROID_LOW_MEMORY_KILLER=y
+# CONFIG_ANDROID is not set
+# CONFIG_ANDROID_BINDER_IPC is not set
+# CONFIG_ANDROID_LOGGER is not set
+# CONFIG_ANDROID_RAM_CONSOLE is not set
+# CONFIG_ANDROID_TIMED_GPIO is not set
+# CONFIG_ANDROID_LOW_MEMORY_KILLER is not set
 # CONFIG_ANDROID_WAKELOCK is not set
-CONFIG_ANDROID_PARANOID_NETWORK=y
+# CONFIG_ANDROID_PARANOID_NETWORK is not set
 
 #
 # File systems
-- 
1.6.5.2

